cmd/multiclusterapp.go

```diff
@@ -722,12 +723,14 @@ func multiClusterAppTemplateInstall(ctx *cli.Context) error {
 	app.Wait = ctx.Bool("helm-wait")
 	app.Timeout = ctx.Int64("helm-timeout")
 
-	_, err = c.ManagementClient.MultiClusterApp.Create(app)
+	app, err = c.ManagementClient.MultiClusterApp.Create(app)
 	if err != nil {
 		return err
 	}
 
-	fmt.Printf("Installing multi-cluster app %q...\n", appName)
+	if app != nil {
```

Comment:
Why the nil check? if there is no error there should be an app. Have you seen differently?